pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -116,7 +118,7 @@ export default defineComponent({
       isDisabled,
       validationMessage,
       requiredField
-    } = useLabeledFormElement(props, emit);
+    } = useLabeledFormElement(props, emit as emitMethodType);
```

Comment:
Adding the emits will change the type of emit passed to the setup method. In this case it will be a union type of `'change' | 'update:value' | 'blur' | 'update:validation'` which surprisingly doesn't match the string type expected by `useLabeledFormElement`. So I extracted the type and then cast appropriately here. I couldn't figure out a type that would work.